src/mol-model-formats/structure/mmcif.ts

```diff
@@ -86,6 +89,74 @@ function structConnFromMmcif(model: Model) {
 }
 StructConn.Provider.formatRegistry.add('mmCIF', structConnFromMmcif);
 
+function indexPairBondsFromMmcif(model: Model) {
```

Comment:
indexPairBondsFromMolstarBondSite? since it is not in the mmcif schema; or maybe just a comment